pySDC/tutorial/step_7/F_2_plot_pySDC_with_Gusto_result.py

```diff
@@ -0,0 +1,205 @@
+def plot_williamson_5(
```

Comment:
I'm slightly concerned that this plotting script could end up getting out of date - but I'm not sure what we do about that! The ones in the Gusto case studies repository will be updated - can you link to that version or were there too many changes?

Reply:
I'm not exactly sure what I'm concerned about! Something may change somewhere...

Reply:
If you think the dependencies of the plotting script are more likely to change than the script running the Williamson 5 test, then I can remove the plotting from pySDC and only put in the link to the Gusto one. 🤷

Reply:
Yeah, I see what you mean... maybe leave it as it is and see if it becomes a problem?